script/travis

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
```

Comment:
I think it's simple enough, but usage examples save time. It took me a second to understand what $1 and $2 were supposed to do because I had to hunt. I sometimes have to get usage help for things because I use so many tools every day that I can mix them up sometimes.